server/router.js

```diff
@@ -26,10 +26,10 @@ module.exports = function(app) {
   app.post("/signin", requireSignin, Authentication.signin);
   app.post("/signup", Authentication.signup);
 
-  app.post("/tasks/add", tasks.addtask);
-  app.get("/tasks/", tasks.getTasks);
+  app.post("/tasks/add", requireAuth, tasks.addtask);
```

Comment:
Is this the only thing that needs to be added to hide the endpoints? I didn't realize it was such a quick thing

Reply:
Yeah. If you go to the top of this file, you can see the middleware that's used to hide the endpoints from unauthenticated users. The only thing is, whenever we need to make requests to these endpoints now, we need to pass the JWT from the `localStorage` as the header. I'm looking into if there's a way to set the header as soon as a user logs in or signs up, but otherwise, we'll have to manually pass in the JWT as the `authorization` header each time.